internal/providers/pluginfw/tfschema/struct_to_schema.go

```diff
@@ -172,18 +174,18 @@ func typeToSchema(v reflect.Value) map[string]AttributeBuilder {
 				elem := typeFieldType
 				sv := reflect.New(elem)
 				nestedScm := typeToSchema(sv)
-				scm[fieldName] = SingleNestedAttributeBuilder{
-					Attributes: nestedScm,
-					Optional:   isOptional,
-					Required:   !isOptional,
-					Computed:   isComputed,
+				scm_block[fieldName] = ListNestedBlockBuilder{
+					NestedObject: nestedScm,
+					Optional:     isOptional,
+					Required:     !isOptional,
+					Computed:     isComputed,
```

Comment:
This is probably not necessary here, because in the generated tfsdk structs there will no longer be any struct types, all will be wrapped as a slice type.